docs/Exports.md

```diff
@@ -31,7 +31,7 @@ Default: `false`
 Set this option to enable parsing of interpolated values for languages such as SCSS. For example:
 `interpolation: { prefix: '@' }` will allow parsing of the interpolated value `@{batman}` which uses `@` as the "prefix". For SCSS one might use `interpolation: { prefix: '#' }`.
 
-##### `interpolation`
+##### `variables`
```

Comment:
@shellscape 
I guess `parentNode` is missing 🤔 
https://github.com/shellscape/postcss-values-parser/blob/ed608973a1fcda642f48a3126695b8c837dfacf1/lib/ValuesParser.js#L24-L32

Reply:
`parentNode` shouldn't be explicitly set by the user, so it was left out on purpose :)